regression/esbmc/github_2338_10/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.c
+--unsigned-overflow-check
```

Comment:
IMHO, this test should not have this flag and it should result in successful verification.
Instead, we should run this testcase once with the `--interval-analysis` flag and once without as stated in #1844 
Instead of `--interval-analysis`, we could also just run the TC once with `--no-simplify`.

Also, could you please move this to `github_1844` so it's clear where the TC originates from?

Reply:
Implemented as suggested.